installers/python-ci/radiasoft-download.sh

```diff
@@ -43,9 +45,13 @@ python_ci_main() {
             set -x
             cd '$d'
             export GITHUB_TOKEN='${GITHUB_TOKEN:-}'
-            if (( $p )); then
+            if (( $p )) || (( $s )); then
```

Comment:
`$p` and `$s` do not need to be coupled. They are implicitly coupled because pykern is installed before sirepo.

Using the array above, this becomes simpler:

```sh
for x in ${p[*]}; do
    pip uninstall -y \$x >& /dev/null || true
    pip install git+https://github.com/radiasoft/\$x.git
done
```

There are a couple of assumptions that are safe, since they are localized:
- `$p` contains words without spaces (Python packages cannot have spaces)
- `$p` is properly ordered (pykern comes before sirepo)